src/main/java/org/fundacion/jala/converter/controller/VideoConverterController.java

```diff
@@ -61,82 +49,15 @@ public String uploadFile(final @RequestParam("file") MultipartFile file,
                              final @RequestParam("height") int height,

```

Comment:
You should write the parameters that can enter within the limit of 120 characters.

Reply:
solved